webapi/src/main/scala/org/knora/webapi/routing/KnoraRoute.scala

```diff
@@ -20,14 +16,20 @@ import scala.concurrent.Future
 import org.knora.webapi.config.AppConfig
 import org.knora.webapi.messages.StringFormatter
 
+import pekko.actor.ActorRef
+import pekko.actor.ActorSystem
+import pekko.http.scaladsl.server.RequestContext
+import pekko.http.scaladsl.server.Route
+import pekko.util.Timeout
+
 /**
  * Data that needs to be passed to each route.
  *
  * @param system    the actor system.
  * @param appActor  the main application actor.
  * @param appConfig the application's configuration.
  */
-case class KnoraRouteData(system: akka.actor.ActorSystem, appActor: akka.actor.ActorRef, appConfig: AppConfig)
+case class KnoraRouteData(system: pekko.actor.ActorSystem, appActor: pekko.actor.ActorRef, appConfig: AppConfig)
```

Comment:
```suggestion
case class KnoraRouteData(system: ActorSystem, appActor: ActorRef, appConfig: AppConfig)
```

should be sufficient, looking at the imports (L19-20 in new line count)